### DISCUSSION TITLE
Centro di costo

### DISCUSSION THREAD

**Discussion by ImpiantoMarketing** [fattureincloud/api#297] [Q&A] [answered]
Salve, stavo provando tramite l'SDK Python a ricevere la lista degli acquisti. Ricevo una lista di [Received Document](https://github.com/fattureincloud/fattureincloud-python-sdk/blob/master/docs/ReceivedDocument.md), tuttavia non c'è un campo relativo al centro di costo. C'è solo un `rc_center` cioè il centro di ricavo. 

Grazie.

**Comment by emajo** [comment, marked as answer]
Ciao @ImpiantoMarketing, in realtà il campo `rc_center` è un campo generico sia per revenue che per cost centers, in base alla tipologa di documento può assumere l'uno o l'altro significato.

Grazie,
Emanele.

**Reply by ImpiantoMarketing** [reply]
> Ciao @ImpiantoMarketing, in realtà il campo `rc_center` è un campo generico sia per revenue che per cost centers, in base alla tipologa di documento può assumere l'uno o l'altro significato.
> 
> Grazie, Emanele.

Ciao @emajo, grazie per la risposta. Tuttavia, utilizzando l'endpoint `GET /c/{company_id}/received_documents` con `type=expense`, il campo `rc_cost` non è mai avvalorato nonostante io lo veda correttamente avvolrato nella UI del sito web per gli acquisti relativi agli ID restituiti dall'endpoint.

Grazie ancora.

**Reply by emajo** [reply]
@ImpiantoMarketing credo tu non stia passando il parametro `fieldset=detailed` e quindi vengono restituiti solo una parte dei campi, confermi? 
[Qui](https://developers.fattureincloud.it/docs/basics/additional-capabilities/customize-response) puoi trovare la documentazione sui fieldset.

**Reply by ImpiantoMarketing** [reply]
> @ImpiantoMarketing credo tu non stia passando il parametro `fieldset=detailed` e quindi vengono restituiti solo una parte dei campi, confermi? [Qui](https://developers.fattureincloud.it/docs/basics/additional-capabilities/customize-response) puoi trovare la documentazione sui fieldset.

Corretto, grazie mille per l'aiuto.
